CRM/Upgrade/Form.php

```diff
@@ -49,6 +49,9 @@ class CRM_Upgrade_Form extends CRM_Core_Form {
 
   /**
    * Minimum php version required to run (equal to or lower than the minimum install version)
+   *
+   * Even though 5.6 is no longer supported, this value is left here for a while
+   * so as not to block stragglers from upgrading.
```

Comment:
:+1: on this.

Tangentially... there are actually several "installers" (i.e. the typical D7/WP web-installer; the drush installer; the Joomla installer; the `civicrm_install()` bash script) which enforce different constraints, and those can be viewed as "stragglers" of another sort. We should eventually fix those too, but *not* by playing whack-a-mole specifically on the PHP version check. Rather, we should push forward on [consolidating the installer code](https://civicrm.org/blog/totten/developers-revising-the-civicrm-installer-library-cli-wordpress) so that it's easier to maintain the installer logic generally.

But that's all tangential.

This PR seems like a fair and simple way to phase-in the elevated requirement.